apps/smart-contracts/core/test/ArbitrageTrading.test.ts

```diff
@@ -28,16 +40,29 @@ describe('=> Functional Testing with Arbitrage Trades', () => {
   let create2Deployer: Create2Deployer
   let univ3Factory: UniswapV3Factory
   let positionManager: NonfungiblePositionManager
+  let swapRouter: SwapRouter
+  let arbitrageBroker: ArbitrageBroker
   const TEST_NAME_SUFFIX = 'preSPACEX 2000-10000 31-December 2022'
   const TEST_SYMBOL_SUFFIX = 'preSPACEX_2000_10000_31_DEC_22'
   const TEST_FLOOR_PAYOUT = ethers.utils.parseEther('0.2')
   const TEST_CEILING_PAYOUT = ethers.utils.parseEther('0.8')
   const TEST_FLOOR_VAL = BigNumber.from(2000)
   const TEST_CEILING_VAL = BigNumber.from(10000)
   const TEST_EXPIRY = nowPlusMonths(2)
+  const TEST_POSITION_SIZE = parseEther('1')
+  /**
+   * Since L/S tokens will be priced at < 1 Collateral, so supplying
+   * twice the Long and Short amount we plan to buy in Collateral is
+   * more than sufficient, and is not a magic number.
+   */
+  const TEST_BROKER_CAPITAL = TEST_POSITION_SIZE.mul(2)
```

Comment:
given above, should only need:
```suggestion
  const TEST_BROKER_CAPITAL = TEST_POSITION_SIZE
```
?

Reply:
This would be true if L/S tokens were _exactly_ 0.5 `Collateral`, but these tests will price L/S tokens at slightly above or below 0.5. TEST_POSITION_SIZE wouldn't be enough, because it could potentially cost more than TEST_POSITION_SIZE `Collateral` to buy both Long and Short token.
e.g. in the case of a non-profitable `buyAndRedeem` situation, it would cost more than (`TEST_POSITION_SIZE` / 2) `Collateral` to buy Long since Long will be priced at slightly above 0.5 `Collateral`, same with buying Short which will add up to > `TEST_POSITION_SIZE` in `Collateral` being spent.